Documentation/contributing/development/reviewers_committers/review_docs.rst

```diff
@@ -0,0 +1,240 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _review_docs:
+
+************************************
+Reviewing for @cilium/docs-structure
+************************************
+
+What is @cilium/docs-structure?
+===============================
+
+Team `@cilium/docs-structure <docs-structure_team_>`_ is a GitHub team of
+Cilium contributors who are responsible for maintaining the good state of the
+project's documentation, by reviewing Pull Requests (PRs) that update the
+documentation. Each time a non-draft PR touching files owned by the team is
+opened on GitHub, one member of the team is automatically assigned for review.
```

Comment:
Assignment is a GitHub Action, right? So we can use active voice accurately here?
```suggestion
documentation. Each time a non-draft PR touching files owned by the team opens on GitHub, a GitHub Action automatically assigns one member of the team for review.
```

Reply:
It's GitHub code review auto-assignment system, I don't think it's an Action itself (my understanding is that an Action is a block that can be used to build Workflows; and that “GitHub Actions” also designate the whole automation system?). But yeah, I can just use “GitHub” as a subject.